components/wpt-results.html

```diff
@@ -98,6 +104,14 @@
           <span class="path-separator">/</span>
           <a href="{{ part.path }}" on-click="_navigate">{{ part.name }}</a>
         </template>
+
+        <template is="dom-if" if="{{ !isLatest }}">
+          <br /><br />
+          <section class="info">
+            Showing results for run <b>{{ sha }}</b>.
+            <small style='float: right;'><a href='?run=latest'>View latest run</a></small>
```

Comment:
Could the inline style be moved up to the stylesheet? It is small and convenient here but I feel like allowing inline styles is a slippery slope.

Reply:
Done.